index.js

```diff
@@ -429,34 +445,6 @@ class ClientServiceBase {
     );
   }
 
-  /**
-   * @param {LedgerValidationRequest} request
-   * @return {Promise<LedgerValidationResult>}
-   * @throws {ClientError|Error}
-   */
-  async _validateLedger(request) {
-    let promises;
-    const ledgerPromise = this._executePromise(
-        this._validateLedgerAsync(this.ledgerClient, request),
-    );
-    if (this._isAuditorEnabled()) {
-      const auditorPromise = this._executePromise(
-          this._validateLedgerAsync(this.auditorClient, request),
-      );
-      promises = [ledgerPromise, auditorPromise];
-    } else {
-      promises = [ledgerPromise];
-    }
-
-    return Promise.all(promises)
-        .then((results) => {
-          return this._validateResult(results[0], results[1]);
-        })
-        .catch((e) => {
-          throw e;
-        });
-  }
-
```

Comment:
Deleted due to https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/85/files#r969210304

We don't need it anymore.